student.rb

```diff
@@ -1,11 +1,12 @@
 require './person'
+require './classroom'
 
 class Student < Person
   attr_reader :classroom
 
-  def initialize(id, classroom, name = 'Unknown', age = nil, parent_permission = true)
+  def initialize(id, _classroom, name = 'Unknown', age = nil, parent_permission = true)
     super(id, name, age, parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
 
```

Comment:
- [ ] @aradradev Great work adding your associations however you forgot to add a `classroom` method that takes in parameter classroom and adds it to the @classroom if it's not already there, to fulfill the 'belongs-to' association between student and classroom.